### ISSUE TITLE
Why is there twice output of yolov8 architecture? Why is the first output GFLOPs different with the second output?

### ISSUE THREAD

**Issue by SwimmingLiu** [ultralytics/ultralytics#4220]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Why is there twice output of yolov8 architecture? Why is the first output GFLOPs different with the second output?
![yolov8_1](https://github.com/ultralytics/ultralytics/assets/53814462/f05e6baf-cb79-4513-84d6-4a6d8535f9a0)
![yolov8_2](https://github.com/ultralytics/ultralytics/assets/53814462/5c20fc16-d493-42dc-8dae-c6c67862fb7b)

**Comment by glenn-jocher** [comment]
@SwimmingLiu hello,

The two outputs of the YOLOv8 architecture you're observing are for two different aspects of the model:

1. The first output, corresponding to the lower GFLOPs, is for the backbone and neck of the YOLOv8 architecture. These parts of the model are responsible for the feature extraction process.

2. The second output, corresponding to the higher GFLOPs, is for the entire YOLOv8 architecture, which includes not only the backbone and neck but also the detection heads. The detection heads are the part of the model that generates the final bounding boxes and class predictions.

The GFLOPs (Giga Floating Point Operations) is a measure of computational demand or complexity. Given that the final detection phase is added in the second output, the GFLOPs are naturally higher because this phase involves more intensive computations.

So, in short, the first output GFLOPs represent the computational load for the feature extraction phase, and the second output GFLOPs represent the computational load for the entire process: feature extraction and final object detection.

I hope this clarifies your question. If you have any more questions or need further clarification, feel free to ask.

Best Regards,
Glenn Jocher

**Comment by SwimmingLiu** [comment]
Thank for your answer!!! I will write the second output into my paper.

**Comment by glenn-jocher** [comment]
@SwimmingLiu you're welcome! I'm glad to hear that the explanation will be useful for your paper. If you run into more questions or need any further clarifications while working on your paper, or when using YOLOv8 in future, don't hesitate to reach out on this repo. The Ultralytics team is here to help. Good luck with your writing!

Best Regards,
Glenn Jocher
